### PR TITLE
Update release notes for v0.5.0

### PR DESCRIPTION
**Pull request by fitzthum**
Add changes that have been made since middle of the release period.

### CODE REVIEW

File: releases/v0.5.0.md [hunk 1]
```diff
@@ -12,20 +17,22 @@ definition of the acronyms used in this document.
 - Process-based isolation is now fully supported with SGX hardware added to enclave-cc CI
 - [KBS Resource URI Scheme](https://github.com/confidential-containers/attestation-agent/blob/main/docs/KBS_URI.md) 
 is published to identify all confidential resources.
-- SEV configuration can be set with annotations
+- Different KBCs now share image encryption format allowing for interchangeable use.
 - Generic Key Broker System (KBS) is now supported. This includes the [KBS](https://github.com/confidential-containers/kbs)
```
Comment (dcmiddle) [inline releases/v0.5.0.md:21]:
```suggestion
- Generic Key Broker System (KBS) is now supported using TDX (support for other TEEs coming soon). This includes the [KBS](https://github.com/confidential-containers/kbs)
```

**Review by dcmiddle — APPROVED** [review]

**Comment by stevenhorsman** [comment]
It think we want to add something like the following statement for peer pods in the what's new:
> Remote hypervisor support added to the CoCo operator, which helps to enable creating containers as ‘peer pods’, either locally, or on Cloud Service Provider Infrastructure. See https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/README.md for more information and installation instructions.

The complication for this is that we'd ideally point at the release version of this readme, rather than staging, but we've not created the release (and want to wait until the attestation-agent and kata-containers have been tagged for 0.5 before we updated peer pods, test it and then release it), so that might be a bit later than this PR wants to be merged?

**Comment by stevenhorsman** [comment]
> > The complication for this is that we'd ideally point at the release version of this readme, rather than staging, but we've not created the release (and want to wait until the attestation-agent and kata-containers have been tagged for 0.5 before we updated peer pods, test it and then release it), so that might be a bit later than this PR wants to be merged?
> 
> No big deal really. This PR is usually one of the last things to be merged before the release so I can update it.

Cool, if is doesn't cause any issues that's great and I'll post the update once we've nailed down the peer pods release (which  probably needs to get added to the release checklist for future) :)

File: releases/v0.5.0.md (hunk 1 above)
Comment (sameo) [reply releases/v0.5.0.md:25]:
```suggestion
- Generic Key Broker System (KBS) is now supported (With TDX and a sample mode, support for other TEEs is coming soon). This includes the [KBS](https://github.com/confidential-containers/kbs)
```

**Review by sameo — COMMENTED** [review]
With one nit to @dcmiddle suggestion. Otherwise LGTM, thanks!

File: releases/v0.5.0.md (hunk 1 above)
Comment (fitzthum) [reply releases/v0.5.0.md:25]: My change differs slightly from @dcmiddle suggestion and it mentions the sample mode.

**Review by sameo — APPROVED** [review]

**Pull request merged**